extending-sassdoc/index.md

```diff
@@ -27,18 +27,18 @@ module.exports.annotations = [];
 ## Schema
 
 Each annotation is an object with a `name` property, a `parse`
-method, and optionnally `resolve` and `default` methods, and
-an `alias` array.
+method, and optionnally `resolve`, `default` and `autofill`
+methods and well as an `alias` array and a `multiple` boolean.
 
 | Key | Type | Description |
 |-----|------|-------------|
 | `name` | string | Name of the annotation. |
-| `parse` | function | Takes the annotation content as parameter and returns the parsed data (can be of any type &mdash; it will be available in the theme as `item.<annotationName>`, as an array since an annotation can be present multiple times). |
+| `parse` | function | Takes the annotation content as parameter and returns the parsed data (can be of any type &mdash; it will be available in the theme as `item.<annotationName>`, as an array if `multiple` is `true`). |
 | `resolve` | function | Called after the raw data is generated, where the whole SassDoc data is being passed (indexed by type and name). You can then modify this object reference as you want to complete your data structure while having access to the whole data. |
-| `default` | function | Returns a default value when &mdash;if ever&mdash; the annotation is not present. |
+| `default` | function | Returns a default value when &mdash; if ever &mdash; the annotation is not present. |
```

Comment:
Should be `&nbsp;` instead of regular space (like in `using-your-own-theme/index.md:127`).